irods/__init__.py

```diff
@@ -41,6 +41,8 @@ def client_logging(flag=True,handler=None):
 CHALLENGE_LEN = 64
 MAX_SQL_ROWS = 256
 DEFAULT_CONNECTION_TIMEOUT = 120
+# https://stackoverflow.com/questions/45704243/value-of-c-pytime-t-in-python
+MAXIMUM_CONNECTION_TIMEOUT = 9223372036.854775
```

Comment:
I'm not convinced we have to keep it so close to the actual MAX of a 64bit signed int of nanoseconds.

We can set it to a much smaller (read: resonable) max.  Perhaps only 100 years(?!).

Reply:
We could also just use the integer part, which would reduce the verbosity of the test... is there a concern about using a maximum limit too close to the system's actual limit?

Reply:
Agreed - integer part seems fine